featuretools_tsfresh_primitives/utils.py

```diff
@@ -0,0 +1,46 @@
+import importlib
+
+import numpy as np
+import pandas as pd
+
+
+def _camel_case(snake_str):
+    words = snake_str.split('_')
+    titled_words = map(str.title, words)
+    return ''.join(titled_words)
+
+
+def _tsfresh_to_featuretools(fc_name):
+    module = importlib.import_module('featuretools_tsfresh_primitives')
+    return getattr(module, _camel_case(fc_name))
+
+
+def primitives_from_fc_settings(fc_settings):
```

Comment:
I know this is being used in the test_primitive test case, but could we add a test case specifically for this function

Reply:
The test for snake case to camel case is a good idea as well but can we have test case for `primitives_from_fc_settings` that passes in a setting and confirms the returned primitives match the expected?

Reply:
I added a test specifically for primitives_from_fc_settings. I also created a test_utils.py to separate this test and the _pascal_case test from the primitives tests.